content/ui-patterns/empty-states.mdx

```diff
@@ -0,0 +1,58 @@
+
+### Secondary text
+This optional text is used to inform the user about the feature in more detail. Secondary text should be brief and non-redundant if possible. From the text, users should  be able to understand the general purpose of the feature and how it may help them accomplish a goal.
+
+### Primary action
+Blankslates can and are encouraged to use one primary action. This button should lead to a feature or component creation flow. Button copy should be keep brief and descriptive. If a button requires further specification, consider adding an Octicon.
+
+### Secondary action
+Secondary actions are optional and are represented by a text link located below the primary action button. A secondary action is used to direct a user to additional content about the feature. This might look like "[Learn more about X](#)" or "[Check out the guide on X](#)" or simply "[Learn more](#)".
+
+### Border
+The border is visible by default and helps define the structure of the blankslate component within the context of the page.
+
+## Variations
+Empty states have multiple variations that can be used in different contexts.
+
+### Pictograms
+[Pictograms](https://ghicons.github.com/) present a great preview of what should be in place of the empty state. Blankslates should default to using a pictogram for graphic elements.
+
+![pictogram blankslate](https://user-images.githubusercontent.com/6846673/62806619-39e39900-baa8-11e9-9de0-5b2f511d63da.png)
+
+### Illustrations
+Illustrations should be used **sparingly** in blankslates - only in cases where it is necessary to evoke emotion from the user. For example, illustrations in a new user experience can help introduce octocats to the user and bring a more personalized experience.
+
+![illustration blankslate](https://user-images.githubusercontent.com/6846673/62806617-394b0280-baa8-11e9-9abc-ccbb76a9f63f.png)
+
+### Compact
+Compact states are a great way to engage and educate a user within Organizations, Settings, and other nested navigation pages. They should replace content that is left-aligned. While the border is optional for all empty states, the Compact variation requires it to better support page structure.
+
+![compact blankslate](https://user-images.githubusercontent.com/6846673/63041086-dbd5fd80-be7b-11e9-8929-3a9c72f53edc.png)
+
+## Content and copy
+Voice and tone are important to making sure users are not only engaged, but also informed about features. Empty states should explain features _in addition_ to conveying intention. Below is a good example of conveying intention in the primary text and using the secondary text to inform.
+
+![code of conduct illustration blankslate](https://user-images.githubusercontent.com/6846673/62806616-394b0280-baa8-11e9-8a5e-45f0c51aaa22.png)
+
+## New user experiences
```

Comment:
Should this be "first time user experiences"? "New user" suggests "new to GitHub", whereas "first time experience" works if they are either new to GitHub or using a specific feature for the first time.